embassy-stm32/src/flash/other.rs

```diff
@@ -0,0 +1,7 @@
+pub trait FlashRegion {
```

Comment:
you can put this in `mod.rs`, that's where all stuff that's shared across versions goes.